scripts/ops/users_github_id_unique_index.sql

```
@@ -0,0 +1,84 @@
+    pg_catalog.hashtextextended('public.users_github_id_key', 0)
+);
+
+-- A failed concurrent build leaves an invalid same-named index. Generate a
+-- concurrent drop only for that recoverable state; valid indexes remain intact.
+SELECT format(
+  'DROP INDEX CONCURRENTLY %I.%I',
+  index_namespace.nspname,
+  idx.relname
+)
+FROM pg_catalog.pg_class AS idx
+JOIN pg_catalog.pg_namespace AS index_namespace
```

SQLFluff (4.2.2)

[error] Join clauses should be fully qualified.

(AM05)